testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt

```diff
@@ -48,12 +49,21 @@ class MiniAppListFragment : BaseFragment(), MiniAppList {
         return binding.root
     }
 
+    override fun onStart() {
+        super.onStart()
+        srl.post {
```

Comment:
I think we can something more descriptive please. `srl` on google gives me "SRL Diagnostics is a diagnostic company" 🎉